woocommerce/woocommerce-android
# [Customer search] Add an event to track add manually button clicked

Closes: #9505


### Description

* Add tracking of the `ORDER_CREATION_CUSTOMER_ADD_MANUALLY_TAPPED` event
* Changes color of the "plus" in the dark mode to match the rest of the app

### Testing instructions

* Use logcat with "track" filter
* Start order creation -> Add customer details -> Click on FAB or "Add customer details" Manually buttons
* Notice `ORDER_CREATION_CUSTOMER_ADD_MANUALLY_TAPPED` tracked


In the dark mode notice white "plus" color on the FAB

### Images/gif

<img width="363" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/aabb1bdc-18f3-4bfb-a153-088da243106a">
